packages/frint-cli/commands/version.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable global-require, import/no-dynamic-require */
+const path = require('path');
+const createApp = require('frint').createApp;
+
+module.exports = createApp({
+  name: 'version',
+
+  providers: [
+    {
+      name: 'summary',
+      useValue: 'Shows current version number of frint-cli',
+    },
+    {
+      name: 'execute',
+      useFactory: function useFactory(deps) {
+        return function execute() {
+          const pkg = JSON.parse(
+            deps.fs.readFileSync(
+              path.resolve(`${__dirname}/../package.json`)
+            )
+          );
+
+          deps.console.log(`v${pkg.version}`);
+        };
+      },
+      deps: [
+        'console',
+        'fs',
```

Comment:
because `path` doesn't have any side-effects like `fs` does. does not cause any harm in tests.

in testing encironment, `fs` is mocked with `memory-fs` package.